bokeh/server/server.py

```diff
@@ -414,20 +425,6 @@ def __init__(self, applications, io_loop=None, http_server_kwargs=None, **kwargs
 
         super().__init__(io_loop, tornado_app, http_server)
 
-    @property
-    def index(self):
-        ''' A path to a Jinja2 template to use for index at "/"
-
-        '''
-        return self._index
-
-    @property
-    def prefix(self):
-        ''' The configured URL prefix to use for all Bokeh server paths.
-
-        '''
-        return self._prefix
-
     @property
     def port(self):
```

Comment:
The one thing I don't love about this work is leaving the `port` and `address` properties instead of deferring to the ones on BaseServer. But if those are are used the log ouptut looks like:
```
2020-05-25 12:49:11,188 Bokeh app running at: http://0.0.0.0:57641/
```
So I think this will have to do for now.
instead of 
```
2020-05-25 12:49:11,188 Bokeh app running at: http://localhost:57641/
`